pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -47,6 +47,28 @@ workflow RNAWithUMIsPipeline {
 		File exonBedFile
 	}
 
+    parameter_meta {
+       bam: "Read group-specific unmapped BAM file;  alternatively, paired-end FASTQ files (the `r1_fastq` and `r2_fastq` inputs) may be used"
+       r1_fastq: "Read 1 FASTQ file; alternatively, the unmapped bam file (`bam` input) may be used as input"
+       r2_fastq: "Read 2 FASTQ file; alternatively, the unmapped bam file (`bam` input) may be used as input"
+       read1Structure: "String describing how the bases in a sequencing run should be allocated into logical reads for read 1"
+       read2Structure: "String describing how the bases in a sequencing run should be allocated into logical reads for read 2"
+       starIndex: "Index file used for STAR aligner"
+       output_basename: "String used as a prefix in workflow output files"
+       gtf: "The gene annotation file (GTF) used in the rnaseq2 task"
+       platform: "String used to describe the sequencing platform; only required when using FASTQ files as input"
+       library_name: "String used to describe the library; only required when using FASTQ files as input"
+       platform_unit: "String used to describe the platform unit; only required when using FASTQ files as input"
+       read_group_name: "String used to describe the read group name; only required when using FASTQ files as input"
+       sequencing_center: "String used to describe the sequencing center; only required when using FASTQ files as input; default is set to “BI”"
+       ref: "FASTA file used for collecting metrics"
```

Comment:
We should have a standard description for the reference files. @ekiernan or @kayleemathews might be able to suggest these? (and vet the parameter_meta section).

Reply:
I don't know if we do but these are the descriptions we currently have in our draft of the pipeline documentation.

Reply:
We don't have current standard descriptions for the reference files, but this is a good idea where possible. I think descriptions should maybe also include the tool that calls it for clarity.

Including the tool means the descriptions won't be completely standardized as the files are used differently across pipelines. For example, the starIndex is being used here for alignment with STAR, whereas the ref is the FASTA file being used for calculating metrics with Picard.